front/app/containers/IdeasNewPage/IdeasNewSurveyForm/index.tsx

```diff
@@ -213,11 +218,8 @@ const IdeasNewSurveyForm = ({ project }: Props) => {
   return (
     <>
       <IdeasNewMeta isSurvey={true} />
-      <PageContainer id="e2e-idea-new-page" overflow="hidden">
-        {!loadingDraftIdea &&
-        schema &&
-        uiSchema &&
-        participationMethodConfig ? (
+      <main id="e2e-idea-new-page">
```

Comment:
I'll also still look at this being in a modal, which is not ideal I believe. We could check with AnySurfer.